recipes/pantab/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
```

Comment:
The license file is in the source distribution so you should remove this manually packaged one.